lib/ecto/soft_delete_repo.ex

```diff
@@ -26,6 +26,21 @@ defmodule Ecto.SoftDelete.Repo do
   """
   @callback soft_delete_all(queryable :: Ecto.Queryable.t()) :: {integer, nil | [term]}
 
+  @doc """
+  Soft restores all entries matching the given query.
+
+  It returns a tuple containing the number of entries and any returned
+  result as second element. The second element is `nil` by default
+  unless a `select` is supplied in the update query.
+
+  ## Examples
+
+      MyRepo.soft_delete_all(Post)
+      from(p in Post, where: p.id < 10) |> MyRepo.soft_delete_all()
```

Comment:
the example should be using `soft_restore_all` and not `soft_delete_all`

Reply:
Good catch, I hadn't noticed